app/views/DR/Home.js

```diff
@@ -0,0 +1,219 @@
+          screen: 'Noticias',
+          params: {
+            screen: 'NewsNavigation',
+            params: {
+              screen: 'Boletines',
+            },
+          },
+        });
+      }
+    }
+  };
+
+  render() {
+    const date = moment(new Date(), 'DD/MM/YYYY').format('MMMM YYYY');
+    const {
+      props: { navigation },
+      state: { cases, deaths, recovered, todayCases, refreshing },
+    } = this;
+
+    return (
+      <View style={stylesSavePath.settingsContainer}>
+          
+        <View
+          tyle={styles.scrollContainer}
+        >
+          <View style={styles.mainHeader} />
+          <View style={{ margin: wp('2%') }} >
+            <View style={styles.HeaderView}>
+              <View style={styles.rowAndCenter}>
+                <Left>
+                  <Text style={[styles.text, { color: '#fff' }]}>
+                    {date[0].toUpperCase() + date.slice(1)}
+                  </Text>
+                </Left>
+              </View>
+              <Text style={styles.headerText}>COVID-RD</Text>
+            </View>
+
+            <View style={styles.marginAndAlign}>
+              <View style={styles.marginAndAlign}>
+                <View style={styles.actualSituationContent}>
+                  <Text style={[styles.subtitles, { alignSelf: 'center' }]}>
+                    Situación actual
+                  </Text>
+                </View>
+              </View>
+              <View style={styles.actualSituationContainer}>
+                <TouchableOpacity
+                  onPress={() => navigation.navigate('map-cities')}
+                >
+                  <Card style={styles.infoCards}>
+                    <Text style={[styles.dataText]}>{cases}</Text>
+                    <Text style={styles.text}>Positivos</Text>
+                  </Card>
+                </TouchableOpacity>
+                <TouchableOpacity
+                  onPress={() => navigation.navigate('map-cities')}
+                >
+                  <Card style={styles.infoCards}>
+                    <Text
+                      style={[
+                        styles.dataText,
+                        { color: Colors.buttonLightText },
+                      ]}
+                    >
+                      {deaths}
+                    </Text>
+                    <Text style={styles.text}>Fallecidos</Text>
+                  </Card>
+                </TouchableOpacity>
+                <TouchableOpacity
+                  onPress={() => navigation.navigate('map-cities')}
+                >
+                  <Card style={styles.infoCards}>
+                    <Text style={[styles.dataText, { color: Colors.green }]}>
+                      {recovered}
+                    </Text>
+                    <Text style={styles.text}>Recuperados</Text>
+                  </Card>
+                </TouchableOpacity>
+                <TouchableOpacity
+                  onPress={() => navigation.navigate('map-cities')}
```

Comment:
Since we don't have this Screen, Could we remove the onPress?

Reply:
done